newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -8,10 +8,22 @@ import BehaviorsEditor from '../BehaviorsEditor';
 import { Tabs, Tab } from 'material-ui/Tabs';
 import { withSerializableObject } from '../Utils/SerializableObjectEditorContainer';
 
+import SemiControlledTextField from '../UI/SemiControlledTextField';
+
 const styles = {
   titleContainer: {
     padding: 0,
   },
+  container: {
```

Comment:
See my comments below. I'm almost sure we can go and do it entirely without styling using components from `Grid.js`.
The goal of having no styling at all is ensuring a super consistent interface through the whole software. In particular, margins are provided by these components.
It's also making the code more maintainable and tweakable. If you want to try slightly larger/smaller margins, just update the margins in a single file. :) Good bye fragile css rules that are never properly updated 😁

Reply:
xD reactjs is so cool. The less we have to write css the better imo